spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/batch/SimpleJobService.java

```diff
@@ -162,27 +162,28 @@ public JobExecution restart(Long jobExecutionId, JobParameters params) throws No
 
 		JobExecution jobExecution;
 
-		if (jsrJobOperator != null) {
-			if (params != null) {
-				jobExecution = new JobExecution(jsrJobOperator.restart(jobExecutionId, params.toProperties()));
-			} else {
-				jobExecution = new JobExecution(jsrJobOperator.restart(jobExecutionId, new Properties()));
-			}
-		} else {
-			throw new NoSuchJobException(String.format("Can't find job associated with job execution id %s to restart",
-					String.valueOf(jobExecutionId)));
-		}
+        try {
+            jobExecution = new JobExecution(jobOperator.restart(jobExecutionId.longValue()));
+        }
+		  catch (JobParametersInvalidException | JobRestartException | NoSuchJobExecutionException |
```

Comment:
Are these 4 exceptions of a common type? Should we just `catch (Exception ex)` instead?